ShareX/Program.cs

```diff
@@ -233,7 +233,19 @@ public static string ScreenshotsFolder
 
         public static string ToolsFolder => Path.Combine(PersonalFolder, "Tools");
         public static string ScreenRecorderCacheFilePath => Path.Combine(PersonalFolder, "ScreenRecorder.avi");
-        public static string DefaultFFmpegFilePath => Path.Combine(ToolsFolder, "ffmpeg.exe");
+        public static string DefaultFFmpegFilePath
+        {
+            get
+            {
+                string ffmpegFilePath = Helpers.GetAbsolutePath("Tools\\ffmpeg.exe");
+                if (!File.Exists(ffmpegFilePath))
```

Comment:
This seems uselessly complex.

```cs
if (File.Exists(ffmpegFilePath))
{
    return ffmpegFilePath;
}
else
{
    return Path.Combine(ToolsFolder, "ffmpeg.exe");
}
```

Reply:
Well I think this is purely opinion based - I was following the rule that method/function should have only 1 return statement (https://stackoverflow.com/a/866092/1155121).